### PR TITLE
4.x lower case template folders

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

**Comment by markstory** [comment]
Changes look good to me, but travis is sad.

**Comment by ADmad** [comment]
Hmm.. I don't get any failure locally.

**Comment by ADmad** [comment]
For some reason the folder for the cell's template file isn't being creating on travis and since our `File` class is so very wonderful it merrily carries along even though `File::pwd()` is unable to set proper value for `File::$path` property, instead of throwing an exception.

File: src/View/BakeView.php [hunk 1]
```diff
@@ -26,6 +26,13 @@ class BakeView extends TwigView
     use ConventionsTrait;
     use InstanceConfigTrait;
 
+    /**
+     * Folder containing bake tempalates.
```
Comment (dereuromark) [inline src/View/BakeView.php]: typo: templates

**Pull request merged**